layers/+lang/python/packages.el

```diff
@@ -325,10 +325,7 @@
         "hH" 'pylookup-lookup))
     :config
     (progn
-      (let ((dir (configuration-layer/get-layer-local-dir 'python)))
-        (setq pylookup-dir (concat dir "pylookup/")
-              pylookup-program (concat pylookup-dir "pylookup.py")
-              pylookup-db-file (concat pylookup-dir "pylookup.db")))
+      (setq pylookup-db-file (concat pylookup-root "pylookup.db"))
       (setq pylookup-completing-read 'completing-read))))
```

Comment:
Just using pylookup-root will not work, for quelpa will not copy the python script to the respective elpa directory.
Instead you need to set the root like this:
```elisp
(setq pylookup-root (expand-file-name "pylookup" quelpa-build-dir))
```
Also setting these variables should be moved into the init section of the package.

Reply:
How about copy the `pylookup.py` from quelpa directory to the installation folder?

Reply:
@sunlin7 
Setting the variable is cheaper than copying the file, and it has no side effect.
The real issue currently is the Python 3 compatibility.

Reply:
@lebensterben  yes, agree, setting the `pylookup.py` path to quelpa directory is cheaper than copy it.
Let's focus on high priority first. 
And copying script file from source folder to destination folder is optimize for local package installization, it's on low priority. 
Thanks